server/settings/tools.py

```diff
@@ -65,6 +66,15 @@ class CreatorToolModel(BaseSettingsModel):
             title="Create Smart Select"
         )
     )
+    # TODO: change to False in next releases
+    use_legacy_product_names_for_renders: bool = SettingsField(
+        True,
+        title="Use legacy product names for renders",
+        description="Use product naming templates for renders. "
+                    "This is for backwards compatibility enabled by default."
+                    "When enabled, it will ignore any templates for renders "
+                    "that are set in the product name profiles.")
```

Comment:
This description seems a bit awkward. First it states "use product naming templates for renders." but if this is enabled, it does not do that, but it does the opposite... 

So the description is actually "Ignore product naming templates for renders but use a hardcoded template. This is for backwards compatibility..." etc. ?